opl/incquery.py

```diff
@@ -80,6 +80,24 @@ class IncQueryProject:
     '''
     def __init__(self, server: str, username: str, password: str, org: str=None, project: str=None, ref: str=None, compartment: str=None, patterns: Hash={}):
 
+        # parse server iri
+        du_iqs = urlparse(server)
+
+        # instantiate iqs_client API client
+        self._y_incquery = iqs_client.ApiClient(iqs_client.Configuration(
+            host=du_iqs.scheme + '://' + du_iqs.netloc + '/api',
+            username=username,
+            password=password,
+        ))
+
+        # instantiate API groups
+        self._y_incquery_queries = iqs_client.QueriesApi(self._y_incquery)
+        self._y_incquery_query_execution = iqs_client.QueryExecutionApi(self._y_incquery)
+        self._y_incquery_in_memory = iqs_client.InMemoryIndexApi(self._y_incquery)
+        self._y_incquery_persistent = iqs_client.PersistentIndexApi(self._y_incquery)
+        self._y_incquery_mms_repo = iqs_client.MmsRepositoryApi(self._y_incquery)
+        self._y_incquery_demo = iqs_client.DemoApi(self._y_incquery)
+
```

Comment:
Initialize the _y_incquery_persistent API before referencing it in _latest_commit